src/Domain/Account/AccountRepositoryInterface.php

```diff
@@ -6,6 +6,9 @@
 
 use PhpFinance\DoubleEntry\Domain\Account\Exception\AccountNotFoundException;
 
+/**
+ * Use {@see AccountManager} instead of repository directly usage.
```

Comment:
```suggestion
 * @internal Use {@see AccountManager} instead of repository direct usage.
```

Reply:
Implementations of this interface will be in separate packages. We don't can make this interface internal.

Reply:
`directly` to `direct` — done